workshops/views.py

```diff
@@ -486,6 +513,32 @@ def membership_create(request, org_domain):
     return render(request, 'workshops/generic_form.html', context)
 
 
+class MembershipCreate(OnlyForAdminsMixin, PermissionRequiredMixin,
+                 CreateViewContext):
+    permission_required = 'workshops.add_membership'
```

Comment:
After looking at other views, I've changed my mind. I don't think we need `change_organization` permission. @pbanaszkiewicz could you confirm that?